lib/workers/bin/provider_worker

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+
+$stdout.sync = true
```

Comment:
NOTE this is here because I noticed that the log line right before the exec wasn't being flushed before the process address space was overwritten so it wasn't making it to the console